english/covid-19/3/3-CA.md

```diff
@@ -0,0 +1 @@
+You are now a bat 👤. You realize that the serum was the first COVID-19 antidote, turning you into a bat to help fight off the virus 😧.
```

Comment:
Hi @gala-m,
Please add an option page

Reply:
Sure thing! let me know if the new options are acceptable